workflows/prognostic_c48_run/tests/test_fv3fit_transformer.py

```diff
@@ -66,7 +89,44 @@ def add_one_to_temperature():
     regression_state(out, regtest)
 
 
-def test_multimodel_adapter(state, tmpdir_factory):
+def test_prediction_inverter():
+    inverter = PredictionInverter(
+        {
+            Q1: MLOutputApplier(TEMP, "tendency"),
+            Q2: MLOutputApplier(SPHUM, "tendency"),
+            NUDGING: MLOutputApplier(TEMP, "tendency"),
+            PRECIP_PRED: MLOutputApplier(PRECIP, "state"),
+        }
+    )
+    prediction = {
+        Q1: xr.DataArray(np.full(NZ, 1 / 86400), dims=("z")),
+        NUDGING: xr.DataArray(np.full(NZ, 1 / 86400), dims=("z")),
+        Q2: xr.DataArray(np.full(NZ, 1 / 86400 / 1000), dims=("z")),
+        PRECIP_PRED: xr.DataArray(np.full(NZ, 1 / 86400), dims=("z")),
+    }
+    inverted_prediction = inverter.invert(prediction)
+    assert TEMP in inverted_prediction
+    assert PRECIP in inverted_prediction
+    xr.testing.assert_allclose(
+        inverted_prediction[TEMP], xr.DataArray(np.full(NZ, 2 / 86400), dims=("z"))
+    )
+
+
+def test_prediction_inverter_raises_multiple_state_update_error():
+    with pytest.raises(ValueError):
+        PredictionInverter(
+            {Q1: MLOutputApplier(TEMP, "state"), Q2: MLOutputApplier(TEMP, "state")}
```

Comment:
Can you avoid the globals here:
```suggestion
            {"q1": MLOutputApplier("t", "state"), "q2": MLOutputApplier("t", "state")}
```